Small fixes for docs and extensions workflow tests

aalves08 opened the pull request:
<!-- This template is for Devs to give QA details before moving the issue To-Test -->
### Summary
Fixes #
<!-- Define findings related to the feature or bug issue. -->

### Occurred changes and/or fixed issues
<!-- Include information of the changes, including collateral areas which have been affected by this PR as requirement or for convenience. -->
- small fix extensions workflow tests by using the correct repo to test with, since the script to get the versions is already using `elemental-ui` repo
- update docs for extensions (recommended node version + update annotation recommendation in 2.10 migration docs)

### Technical notes summary
<!-- Outline technical changes which may pass unobserved or may help to understand the process of solving the issue -->

### Areas or cases that should be tested
<!-- Areas that should be tested can include Airgap checks, Rancher upgrades, K8s upgrade, etc. -->
<!-- Which browser did you use for local testing? The reviewer should test with a different browser. -->
<!-- Add missing steps or rewrite them if have been missed or to complement existing information. This should define a clear way to reproduce it and not an approximation. -->

### Areas which could experience regressions
<!-- Create a detailed list of areas to be analyzed which may be affected by the changes, which would require a prior research to avoid regressions. -->

### Screenshot/Video
<!-- Attach screenshot or video of the changes and eventual comparison if you find it necessary -->

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone <!-- The milestone should automatically be assigned if the linked issue has one, but does not always happen (incorrectly linked, issue has no milestone, etc) -->
- [x] The PR template has been filled out
- [x] The PR has been self reviewed <!-- There are no TODOs, no incorrect files in the PR, all the required files are there, no commented out code, etc-->
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

jtomasek reviewed (commented):
Should the annotation here be also updated? https://github.com/rancher/dashboard/blob/master/creators/extension/migrate/tasks/packageUpdates.js#L217

aalves08 commented:
> Should the annotation here be also updated? https://github.com/rancher/dashboard/blob/master/creators/extension/migrate/tasks/packageUpdates.js#L217

Nice catch @jtomasek ... I updated the creator but forgot about the migrate script. Thanks! 💪

torchiaf reviewed (approved):
LGTM
